test/integration/targets/setup_wildfly_server/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+remote_tmp_dir: /tmp/wildfly_tmp
```

Comment:
Sorry, I guess I forgot to mention that you should get `remote_tmp_dir` by adding a meta dependency on the `setup_remote_tmp_dir` role, which will take care of creating it and deleting it.

Reply:
Nps, will see :)

Reply:
changed, convenient, thanks! added to my check list